src/Expression/Boolean/Orx.php

```diff
@@ -0,0 +1,43 @@
+<?php
+declare(strict_types=1);
+
+namespace Arkitect\Expression\Boolean;
+
+use Arkitect\Analyzer\ClassDescription;
+use Arkitect\Expression\Description;
+use Arkitect\Expression\Expression;
+use Arkitect\Rules\Violation;
+use Arkitect\Rules\ViolationMessage;
+use Arkitect\Rules\Violations;
+
+final class Orx implements Expression
+{
+    /** @var Expression[] */
+    private $expressions;
+
+    public function __construct(array $expressions)
+    {
+        $this->expressions = $expressions;
+    }
```

Comment:
**suggestion**: use variable length argument lists.
```suggestion
    public function __construct(...$expressions)
    {
        $this->expressions = $expressions;
    }
```
```php
new Orx(new Extend('My\BaseClass'), new Extend('Your\OtherClass'));
```
because in phparkiteck upgrade in ^7.4 it will make typing easier:
```suggestion
    public function __construct(Expression ...$expressions)
    {
        $this->expressions = $expressions;
    }
```
It will still be possible to use it with an array such as :
```php
new Orx(...[new Extend('My\BaseClass'), new Extend('Your\OtherClass')]);
```